starter-code/css/main.css

```diff
@@ -1,10 +1,79 @@
-/*
-Colors to use:
-dark blue: #06365f
-light blue: #1c5380
-*/
-
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
 }
+
+.background {
+  height: 300px;
+  background-color: #06365f;
+}
+
+.box {
+  display: flex;
```

Comment:
you were supposed to use float ( but flex is better :joy::joy::joy::joy::joy: it's ok)